enterprise/frontend/src/embedding-sdk/components/public/MetabotQuestion/QuestionDetails.tsx

```diff
@@ -0,0 +1,82 @@
+import { t } from "ttag";
+
+import { useBreakoutData } from "embedding-sdk/components/private/InteractiveQuestion/components/Breakout/use-breakout-data";
+import { useFilterData } from "embedding-sdk/components/private/InteractiveQuestion/components/Filter/hooks/use-filter-data";
+import { useSummarizeData } from "embedding-sdk/components/private/InteractiveQuestion/components/Summarize/use-summarize-data";
+import { useInteractiveQuestionContext } from "embedding-sdk/components/private/InteractiveQuestion/context";
+import { isNotNull } from "metabase/lib/types";
+import { Text } from "metabase/ui";
+import * as Lib from "metabase-lib";
+import type Question from "metabase-lib/v1/Question";
+
+export function QuestionDetails() {
+  const { question } = useInteractiveQuestionContext();
+  const tableSection = getTableSection(question);
```

Comment:
Should we add the table or model suffix according to the design? e.g. "Orders table"

![CleanShot 2568-04-12 at 03 15 16@2x](https://github.com/user-attachments/assets/c5252882-fcfa-406d-83a4-fd3ac460fec0)